### PR TITLE
Tempus 851 logo change

### PR DESCRIPTION
**Pull request by NandiniMGawade**
Thank you for submitting a contribution to Tempus.

In order to streamline the review of the contribution please
ensure the following steps have been taken:

### For all changes:
- [ ] Is there a Waffle ticket associated with this PR? Is it referenced 
     in the commit message?

- [ ] Does your PR title start with Tempus-XXXX where XXXX is the waffle number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically dev)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn clean install at the root Tempus folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] Have you added documentation for any UI or API related changes to the /docs folder

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: application/src/main/java/com/hashmapinc/server/controller/UserSettingsController.java [hunk 1]
```diff
@@ -167,4 +175,17 @@ public Logo getLogo() throws TempusException  {
         }
     }
 
+    @GetMapping(value = "/settings/{tenantId}/logo")
+    @ResponseStatus(value = HttpStatus.OK)
+    public String userLogo(@PathVariable("tenantId") String strTenantId) throws TempusException {
+        checkParameter("tenantId", strTenantId);
+        try {
+            TenantId tenantId = new TenantId(toUUID(strTenantId));
+            log.info("tenant in controller{}",tenantId);
```
Comment (akshaymhetre) [inline application/src/main/java/com/hashmapinc/server/controller/UserSettingsController.java]: remove logs

File: dao/src/main/java/com/hashmapinc/server/dao/tenant/TenantServiceImpl.java [hunk 2]
```diff
@@ -185,4 +189,13 @@ private void createGroupForTenant(String title, TenantId savedTenantId) {
         customerGroup.setPolicies(Collections.singletonList(DataConstants.TENANT_ADMIN_DEFAULT_PERMISSION));
         customerGroupService.saveCustomerGroup(customerGroup);
     }
+
+    @Override
+    public JsonObject findLogoByTenantId(TenantId tenantId) {
```
Comment (akshaymhetre) [inline dao/src/main/java/com/hashmapinc/server/dao/tenant/TenantServiceImpl.java]: Change Return type to String

File: dao/src/main/resources/sql/hsql/schema.sql [hunk 3]
```diff
@@ -207,7 +207,8 @@ CREATE TABLE IF NOT EXISTS tenant (
     search_text varchar(255),
     state varchar(255),
     title varchar(255),
-    zip varchar(255)
+    zip varchar(255),
+    logo VARCHAR
```
Comment (akshaymhetre) [inline dao/src/main/resources/sql/hsql/schema.sql]: Please add length to VARCHAR for logo.

File: ui/src/app/api/user.service.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ export default angular.module('tempus.api.user', [tempusApiLogin,
     .name;
 
 /*@ngInject*/
-function UserService($http, $q, $rootScope, adminService, dashboardService, loginService, toast, store, jwtHelper, $translate, $state, $location) {
+function UserService($http, $q, $log, $rootScope, adminService, dashboardService, loginService, toast, store, jwtHelper, $translate, $state, $location) {
```
Comment (akshaymhetre) [inline ui/src/app/api/user.service.js]: Remove $log

**Comment by akshay24871** [comment]
I have tested. It looks good to me.

**Pull request merged**